webapp/stores/emoji_store.jsx

```diff
@@ -148,8 +149,7 @@ class EmojiStore extends EventEmitter {
 
     getEmojiImageUrl(emoji) {
         if (emoji.id) {
-            // must match Client.getCustomEmojiImageUrl
-            return `/api/v3/emoji/${emoji.id}`;
+            return Client.getCustomEmojiImageUrl(emoji.id);
```

Comment:
Actually I don't think there is any resonable criteria which cleary distinguish portable code and browser-dependent code unless you provide modules for the respective environments. Browsers provide more than UI and such features can be required also in modules which needs to be testes as they are required by `web_client.jsx`.